### PR TITLE
moves amanda to alumni

### PR DESCRIPTION
**Pull request by samgdotson**
## Summary of changes
Moves @abachma2 from "People" to "Alumni."

### CODE REVIEW

File: _data/alumni.yml [hunk 1]
```diff
@@ -60,6 +60,22 @@
       icon: github-alt
       name: Github
 
+- name: Amanda Bachmann
+  pic: /img/people/bachmanna.png
+  dates: 2020-present
```
Comment (abachma2) [inline _data/alumni.yml:65]:
```suggestion
  dates: 2020-2023
```

**Review by nsryan2 — APPROVED** [review]
🥲 on to bigger and better things

File: _data/alumni.yml [hunk 2]
```diff
@@ -60,6 +60,22 @@
...
+  role: Graduate Research Assistant
+  description: "Amanda is a PhD student interested in the nuclear fuel cycle and the best ways to implement nonproliferation safeguards."
```
Comment (abachma2) [inline _data/alumni.yml:67]:
```suggestion
  description: "Amanda was a PhD student working on fuel cycle modeling. She received her PhD in 2023, and is no  postdoc at Argonne National Lab."
```

File: _data/alumni.yml [hunk 3]
```diff
@@ -60,6 +60,22 @@
...
+  social:
+      - url: mailto:amandab7@illinois.edu
+        icon: envelope-o
+        name: email
```
Comment (abachma2) [inline _data/alumni.yml:71]:
```suggestion

```

**Review by abachma2 — CHANGES_REQUESTED** [review]
Thanks for taking care of this @samgdotson. I suggested a few tweaks.

File: _data/alumni.yml (hunk 3 above)
Comment (nsryan2) [reply _data/alumni.yml:71]: What is this changing?

File: _data/alumni.yml (hunk 2 above)
Comment (nsryan2) [reply _data/alumni.yml:67]: Should say, "is now a postdoc"?

Comment (abachma2) [reply _data/alumni.yml:67]: Yes. Thank you.

File: _data/alumni.yml (hunk 3 above)
Comment (abachma2) [reply _data/alumni.yml:71]: As far as I can tell I'll lose access to this email 1 year after graduation, so maybe not the best to have that there.

File: _data/alumni.yml (hunk 2 above)
Comment (nsryan2) [reply _data/alumni.yml:67]: Want to link to your PhD?

Comment (abachma2) [reply _data/alumni.yml:67]: My dissertation? I don't think it's live yet on Ideas

Comment (nsryan2) [reply _data/alumni.yml:67]: Ahh, gottcha! I didn't know how that worked, but fair.

File: _data/alumni.yml [hunk 4]
```diff
@@ -60,6 +60,19 @@
       icon: github-alt
       name: Github
 
+- name: Amanda Bachmann
+  pic: /img/people/bachmanna.png
+  dates: 2020-2023
+  role: Graduate Research Assistant
+  description: "Amanda was a PhD student working on fuel cycle modeling. She received her PhD in 2023, and is no  postdoc at Argonne National Lab."
```
Comment (abachma2) [inline _data/alumni.yml:67]:
```suggestion
  description: "Amanda was a PhD student working on fuel cycle modeling. She received her PhD in 2023, and is now a postdoc at Argonne National Laboratory."
```

**Review by abachma2 — COMMENTED** [review]
Sorry, found a small typo.

**Review by abachma2 — APPROVED** [review]
Thanks @samgdotson. Approving and merging.

**Pull request merged**